### PR TITLE
chore(tutorials): fixed punctuation, typos and grammar in tutorials.

### PR DESCRIPTION
**Pull request by Norem80**
## Description
I found some grammar errors in tutorials and fixed them.

### CODE REVIEW

File: docs/tutorial/part-eight/index.md [hunk 1]
```diff
@@ -198,9 +198,9 @@ class Application extends React.Component {
 
 ## Keep making it better
 
-In this section we've shown you a few Gatsby-specific tools to improve your site's performance and prepare to go live.
+In this section, we've shown you a few Gatsby-specific tools to improve your site's performance and prepare to go live.
 
-Lighthouse is a great tool for site improvements and learning -- Continue looking through the detailed feedback it provides and keep making your site better!
+Lighthouse is a great tool for site improvements and learning -- Continue looking through the detailed feedback it provides and keeps making your site better!
```
Comment (sidharthachatterjee) [inline docs/tutorial/part-eight/index.md]: _keep_ seems correct in this instance?

File: docs/tutorial/part-five/index.md [hunk 2]
```diff
@@ -91,7 +91,7 @@ Space</kbd>).
 
 ![filesystem-autocomplete](filesystem-autocomplete.png)
 
-Try adding a number of fields to your query, pressing <kbd>Ctrl + Enter</kbd>
+Try adding a number of fields to your query pressing <kbd>Ctrl + Enter</kbd>
```
Comment (sidharthachatterjee) [inline docs/tutorial/part-five/index.md]: The comma seems correct in this instance?

File: docs/tutorial/part-four/index.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ typora-copy-images-to: ./
 Welcome to Part Four of the tutorial! Halfway through! Hope things are starting
 to feel pretty comfortable 😀
 
-## Recap of first half of the tutorial
+## Recap of the first half of the tutorial
```
Comment (sidharthachatterjee) [inline docs/tutorial/part-four/index.md]: Not necessary in this context, reads okay in my opinion without the _the_

File: docs/tutorial/part-one/index.md [hunk 4]
```diff
@@ -45,7 +45,7 @@ Open the file at `src/pages/index.js`. The code in this file creates a component
 
 <video controls="controls" autoplay="true" loop="true">
   <source type="video/mp4" src="./02-demo-hot-reloading.mp4"></source>
-  <p>Sorry! You browser doesn't support this video.</p>
+  <p>Sorry! Your browser doesn't support this video.</p>
```
Comment (sidharthachatterjee) [inline docs/tutorial/part-one/index.md]: Good catch! 👍

File: docs/tutorial/part-three/index.md [hunk 5]
```diff
@@ -13,7 +13,7 @@ Gatsby plugins are JavaScript packages that help add functionality to a Gatsby s
 
 Layout components are for sections of your site that you want to share across multiple pages. For example, sites will commonly have a layout component with a shared header and footer. Other common things to add to layouts are a sidebar and/or navigation menu. On this page for example, the header at the top is part of gatsbyjs.org's layout component.
 
-Let's dive in to part three.
+Let's dive into part three.
```
Comment (sidharthachatterjee) [inline docs/tutorial/part-three/index.md]: 👍

File: docs/tutorial/part-three/index.md [hunk 6]
```diff
@@ -271,7 +271,7 @@ export default ({ children }) => (
 
 ![with-navigation2](with-navigation2.png)
 
-And there you have it! A three page site with a basic global navigation.
+And there you have it! A three page site with basic global navigation.
```
Comment (sidharthachatterjee) [inline docs/tutorial/part-three/index.md]: Good catch again!

File: docs/tutorial/part-two/index.md [hunk 7]
```diff
@@ -253,7 +253,7 @@ The `console.log(styles)` code will log the resulting import so you can see the
 
 If you compare that to your CSS file, you'll see that each class is now a key in the imported object pointing to a long string e.g. `avatar` points to `src-pages----about-css-modules-module---avatar---2lRF7`. These are the class names CSS Modules generates. They're guaranteed to be unique across your site. And because you have to import them to use the classes, there's never any question about where some CSS is being used.
 
-4. Create a `User` component..
+4. Create a `User` component.
```
Comment (sidharthachatterjee) [inline docs/tutorial/part-two/index.md]: Good eye 👀

File: docs/tutorial/part-eight/index.md (hunk 1 above)
Comment (Norem80) [reply docs/tutorial/part-eight/index.md]: it relates to "it" so it is in the third person.

File: docs/tutorial/part-five/index.md (hunk 2 above)
Comment (Norem80) [reply docs/tutorial/part-five/index.md]: Oh, I must have removed it by accident and didn't catch it later.

File: docs/tutorial/part-eight/index.md (hunk 1 above)
Comment (sidharthachatterjee) [reply docs/tutorial/part-eight/index.md]: The latter half of the sentence (after the and): "keep making your site better" relates to the reader and asks them to _keep_ making their site better by continually looking through the feedback!

File: docs/tutorial/part-five/index.md (hunk 2 above)
Comment (sidharthachatterjee) [reply docs/tutorial/part-five/index.md]: No worries!

File: docs/tutorial/part-eight/index.md (hunk 1 above)
Comment (Norem80) [reply docs/tutorial/part-eight/index.md]: Yeah, that's right, I haven't got it earlier, but now I do. Gonna do a little fix!

**Comment by Norem80** [comment]
I guess it's all right, now.

File: docs/tutorial/part-eight/index.md (hunk 1 above)
Comment (sidharthachatterjee) [reply docs/tutorial/part-eight/index.md]: No worries. Better grammar through collaboration! 🙌

File: docs/tutorial/part-five/index.md [hunk 8]
```diff
@@ -91,7 +91,7 @@ Space</kbd>).
 
 ![filesystem-autocomplete](filesystem-autocomplete.png)
 
-Try adding a number of fields to your query, pressing <kbd>Ctrl + Enter</kbd>
+Try adding a number of fields to your query pressing, <kbd>Ctrl + Enter</kbd>
```
Comment (sidharthachatterjee) [inline docs/tutorial/part-five/index.md]: This seemed correct earlier as well! Sorry for the back and forth!

Comment (Norem80) [reply docs/tutorial/part-five/index.md]: Yeah, no problem.

**Pull request merged**